Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -149,26 +153,36 @@ def keg_contain?(string, keg, ignores, formula_and_runtime_deps_names = nil, arg
         until io.eof?
```

Comment:
I'd be tempted to pull this entire block into a separate function (perhaps in `keg-relocate.rb`?; it seems relatively independent? Np if it's less than I thought.

Reply:
I went ahead and extracted it.

I agree, it makes sense to have it separated. It does has a handful of parameters that need to be passed so take a look at if you think it shouldn't really belong in `keg_relocate` I can certainly move it back to `bottle.rb` in a separate function.